### PR TITLE
allow user to configure jwt connect/read timeout

### PR DESCRIPTION
**Pull request by sophiaso**
Configurable JWT read, connect timeout and size limit

```
azure.activedirectory.jwt-connect-timeout
azure.activedirectory.jwt-read-timeout
azure.activedirectory.jwt-size-limit
```

### CODE REVIEW

**Comment by vipinsaini434** [comment]
It is working fine by changing the time limit.

File: azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java [hunk 1]
```diff
@@ -38,6 +39,12 @@
     @NotEmpty
     private List<String> activeDirectoryGroups;
 
+    private int jwtConnectTimeout = RemoteJWKSet.DEFAULT_HTTP_CONNECT_TIMEOUT; /* milliseconds */
```
Comment (Incarnation-p-lee) [inline azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java]: We can file another PR for `@Getter`.

**Comment by kai18** [comment]
Hi @Incarnation-p-lee @laxmikant4644 . Has this change been deployed to the latest available version on maven?

**Comment by Incarnation-p-lee** [comment]
@kai18 
Thanks for you information. Yes, we have released [Azure Spring boot Starters 2.0.8](https://search.maven.org/artifact/com.microsoft.azure/azure-spring-boot/2.0.8/jar) for spring boot 2.0.x, as well as [Azure Spring boot Starters 2.1.0](https://search.maven.org/artifact/com.microsoft.azure/azure-spring-boot/2.1.0/jar) for spring boot 2.1.x.

**Comment by kai18** [comment]
Ah! Thanks @Incarnation-p-lee . Apparently it was an IDE issue. IntelliJ did not properly index and so couldn't find the properties.

**Comment by Incarnation-p-lee** [comment]
@kai18 
Great! Any other question/issue please feel free to contact us!

**Pull request merged**